### ISSUE TITLE
VideoMode not working

### ISSUE THREAD

**Issue by Mowlawner** [emukidid/Wii64#79]
When I set VideoMode=2 in settings.cfg in sd:/wii64/ and ensure the file is not a .txt file, but a .cfg file, wii64 does not respect my desire to output in 240p. Issue occurs in ricegfx and glidegfx versions, wiivc not tested.

**Comment by saulfabregwiivc** [comment]
@Mowlawner can you try forcing Wii64 to 240p but in arguments?

Add the following arguments between <app> and </app>:
`<arguments>`
`<arg>VideoMode=2</arg>`
`</arguments>`

Tell us if it's working.

**Comment by elias-mills-levvel** [comment]
@saulfabregwiivc sorry for the delay- just for clarity, to which file should I add these lines? settings.cfg?

**Comment by Mowlawner** [comment]
Sorry for the delay- which file am I adding this to? settings.cfg?

**Comment by saulfabregwiivc** [comment]
> Sorry for the delay- which file am I adding this to? settings.cfg?

No. These arguments must be added to meta.xml file. (sorry i forgot to say it)

**Comment by Mowlawner** [comment]
Thank you for clarifying! I just copy and pasted the arguments into the Glide and Rice meta.xml files, but they're both still outputting 480 for me.

**Comment by saulfabregwiivc** [comment]
well, that's another issue. Maybe @emukidid needs to see and fix it

**Comment by leodexe** [comment]
This is probably a dumb question but you made sure your TV does support 480i output?
Arguments in meta.xml are working for me, my TV already outputs 480p but I force it VideoMode=3 which equals 480p so I'm never accidentally booting into 240p

**Comment by Mowlawner** [comment]
@leodexe thanks for the reply! at the time i was using a CRT that was running other emulators on the same wii at 240p with no issue. unfortunately i never found a fix, but for what it's worth i no longer have that tv anyway so i'm no longer in a position to experience that issue anymore
